web/packages/teleterm/src/ui/TopBar/TopBar.tsx

```diff
@@ -49,11 +49,12 @@ export function TopBar() {
 }
 
 const Grid = styled(Flex).attrs({ gap: 3, py: 2, px: 3 })`
-  background: ${props => props.theme.colors.levels.surfaceSecondary};
+  background: ${props => props.theme.colors.levels.surface};
   width: 100%;
   height: 56px;
   align-items: center;
   justify-content: space-between;
+  z-index: 2; // minimally higher z-index than the one defined in StyledTabs, so that its drop-shadow doesn't cover the TopBar 
```

Comment:
Is it possible to manipulate the box-shadow in a way where it doesn't appear at the top of the div so that we don't have to manipulate z-index to fix this?

Reply:
I've tried that, but without much success (and our shadow contains 3 sub-shadows which makes it harder).